azure-pipelines/end-to-end-tests-dir/bundles.ps1

```diff
@@ -0,0 +1,149 @@
+. $PSScriptRoot/../end-to-end-tests-prelude.ps1
+
+if ($IsWindows) {
+    $cache_home = $env:LOCALAPPDATA
+} elseif (Test-Path "env:XDG_CACHE_HOME") {
+    $cache_home = $env:XDG_CACHE_HOME
+} else {
+    $cache_home = "$env:HOME/.cache"
+}
+$bundle = Join-Path $TestingRoot "bundle"
+$manifestdir = Join-Path $TestingRoot "manifest"
+
+# Test classic bundle
+New-Item -ItemType Directory -Force $bundle | Out-Null
+New-Item -ItemType File -Force $bundle/.vcpkg-root | Out-Null
+
+$a = Run-Vcpkg z-print-config `
+    --vcpkg-root=$bundle `
+    --overlay-triplets=$env:VCPKG_ROOT/triplets `
+    --x-scripts-dir=$env:VCPKG_ROOT/scripts
+$a
+Throw-IfFailed
```

Comment:
It doesn't affect it, and I explicitly needed the printed value _before_ the throw in order to diagnose what the throw was caused by (the vcpkg error output, for example). I don't see much use in printing the value below the `Throw-IfFailed` because the output was "correct"

Reply:
`$LastExitCode` is only set by external process calls, which `$a` doesn't do.